subworkflows/local/utils_nfcore_smrnaseq_pipeline/main.nf

```diff
@@ -198,26 +216,52 @@ def genomeExistsError() {
 // Generate methods description for MultiQC
 //
 def toolCitationText() {
-    // TODO nf-core: Optionally add in-text citation tools to this list.
     // Can use ternary operators to dynamically construct based conditions, e.g. params["run_xyz"] ? "Tool (Foo et al. 2023)" : "",
     // Uncomment function in methodsDescriptionText to render in MultiQC report
     def citation_text = [
             "Tools used in the workflow included:",
             "FastQC (Andrews 2010),",
-            "MultiQC (Ewels et al. 2016)",
+            "MultiQC (Ewels et al. 2016),",
```

Comment:
OMG ever since I added this I think this is the first time I've seen someone else use it :star_struck: .

Do all tools get used by default? If some are optional you can also specify each tool with a condition e.g. `params.run_<tool> ? <ref> : ''`

Reply:
Majority gets used all the time, but we can likely update this to be somewhat smarter than this :)